src/viam/examples/modules/example_mlmodel.cpp

```diff
@@ -27,17 +27,18 @@ class ExampleMLModelService : public vs::MLModelService {
     std::shared_ptr<named_tensor_views> infer(const named_tensor_views& inputs) override {
         std::cout << "ExampleMLModelService: recieved `infer` invocation" << std::endl;
 
-        static constexpr std::array<float, 4> location_data = {0.1, 0.1, 0.75, 0.75};
-        static constexpr std::array<float, 1> category_data = {0};
-        static constexpr std::array<float, 1> score_data = {.99};
-        static constexpr std::array<float, 1> num_dets_data = {1};
+        static constexpr std::array<float, 100> location_data = {0.1, 0.1, 0.75, 0.75};
```

Comment:
The changes in this file could be held back until tflite example is ready. All that is happening here is that we are returning real shape metadata below, so the backing data for the tensor actually needs to be the right shape.